.github/workflows/post-release.yml

```diff
@@ -36,19 +36,24 @@ jobs:
         # for this work.
         fetch-depth: 0
         ref: ${{ github.event.repository.default_branch }}
-        token: ${{ secrets[needs.automation.outputs.token-secret-name] }}
+        token: ${{ env.GH_TOKEN }}
 
     - name: Create GitHub Pull Request to update PackageValidationBaselineVersion in projects
       if: |
-        (github.ref_type == 'tag' && !contains(github.ref_name, '-alpha') && !contains(github.ref_name, '-beta') && !contains(github.ref_name, '-rc'))
-        || (inputs.tag && !contains(inputs.tag, '-alpha') && !contains(inputs.tag, '-beta') && !contains(inputs.tag, '-rc'))
+        (github.ref_type == 'tag' && !contains(github.ref_name, '-alpha') && !contains(github.ref_name, '-beta') && !contains(github.ref_name, '-rc')) ||
+        (inputs.tag && !contains(inputs.tag, '-alpha') && !contains(inputs.tag, '-beta') && !contains(inputs.tag, '-rc'))
       shell: pwsh
+      env:
+        BOT_USER_EMAIL: ${{ needs.automation.outputs.email }}
+        BOT_USER_NAME: ${{ needs.automation.outputs.username }}
+        DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
```

Comment:
In .github/workflows/prepare-release.yml it is called TARGET_BRANCH, I think it is better name